internetee/auction_center
# added toggle to switch estonian tax rate

Related to https://github.com/internetee/eis_billing_system/issues/104

**Objective of the Task**
After the New Year, the income tax rate in Estonia changes from 20 to 22 percent. Due to this, it is necessary to modify the variables for the calculation.

**What have I done?**
I have added a new key to the customization.yml file `is_new_vat_rate_available`, which takes a Boolean value - true or false. If true, the total sum is calculated using the new tax rate of 22 percent; if false, the old rate of 20 percent is retained.

**How to Test?**
Try switching these values and check how the overall payment calculation changes. Also, ensure that the correct sum calculation is sent to directo and e-invoice.
